source/includes/administration/_service_connections.md

```diff
@@ -225,6 +225,118 @@ Attributes | &nbsp;
 
+<!-------------------- GET SERVICE DESCRIPTOR -------------------->
+### Retrieve descriptor
+
+`GET /services/descriptors/:pluginType`
+
+```shell
+# Retrieve service descriptor
+
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/api/v2/services/descriptors/gcp/parameters"
+```
+
+> Request body example:
+
+```json
+{
+    "data": {
+        "type": "gcp",
+        "entityDescriptors": [
+            {
+                "type": "externalips",
+                "serviceType": "gcp",
+                "system": false,
+                "operations": {
+                    "release": {
+                        "name": "release",
+                        "serviceType": "gcp",
+                        "immediateAction": false,
+                        "isDownload": false,
+                        "defaultOperationType": "DELETE",
+                        "permissionLevel": "BASE"
+                    },
+                    ...
+                },
+                "costEstimators": [],
+                "viewBuilders": [
+                    {
+                        "serviceType": "gcp",
+                        "view": "LIST",
+                        "defaultView": false,
+                        "defaultSpecificView": false
+                    },
+                    ...
+                ]
+            },
+            ...
+        ],
+        "infraEntityDescriptors": [],
+        "supportsInfra": false,
+        "supportedWidgets": [],
+        "hasPluginEnvironmentRoles": false,
+        "quotaMetricIdentifiers": [
+            "instance.cpu.reserved_cores",
+            ...
+        ],
+        "supportsQuota": true,
+        "supportsEnvironmentMetrics": false,
+        "supportsApiCredentials": false,
+        "supportsPolicies": true,
+        "supportsUsage": true,
+        "supportsPricingV2": true,
+        "supportsCaching": false,
+        "supportsMetric": true,
+        "metricDescriptors": [
+            {
+                "metricIdentifier": "instance.cpu.utilization",
+                "nameLabelKey": "metrics.instances.cpu_utilization.label",
+                "unitLabelKey": "metrics.instances.cpu_utilization.unit",
+                "entityType": "instances",
+                "resourceMetricType": "GAUGE"
+            },
+            ...
+        ],
+        "entityRelationships": {
+            "vpntunnels": [
+                {
+                    "entityType": "networks",
+                    "relationshipType": "PARENT"
+                }
+            ],
+            ...
+        },
+        "extensions": {
+            "ksdk": {}
+        }
+    }
+}
+```
+
+Attributes                                  | &nbsp;
+------------------------------------------- | --------------------------------------------------------
+`data`<br/>*Array[object]*                  | An array of service descriptors, each with the following properties:
```

Comment:
I dont think we usually document the data field as part of the attribute I may be wrong.

Also, in the payload it does not use the array notation but the object notation. Is that on purpose ?

Reply:
That's a good point, let me address that